godel/config/check-plugin.yml

```diff
@@ -4,3 +4,4 @@ checks:
     - value: should have comment or be unexported
     - value: or a comment on this block
     - value: struct field Uid should be UID
+    - value: should be consistent with previous receiver name
```

Comment:
This check is failing because the plugin is not working with generics and thinks that `graph[T]` is the same as `node[T]`.